File: AreaTriangulo.kt
fun main(args: Array<String>){
  var b : Int
  var h : Int
  print("Ingrese base: ")
  b = readLine()!!.toInt()
  print("Ingrese altura: ")
  h = readLine()!!.toInt()
  var area : Int
  area = b*h/2
  println("El área del triangulo es: $area")
}

File: AreaTriangulo.scala
import scala.io.StdIn.readLine 
object AreaTSC { 
  def main(args: Array[String]) = { 
    //Declaracion de Variables y Otros 
    println("Ejercicio 01: Area Triangulo") 
    //Datos de entrada 
    println("Ingrese Base:") 
    val b = readLine().toInt 
    println("Ingrese Altura:") 
    val h = readLine().toInt 
    //Proceso 
    val area=(b*h)/2 
    //Datos de Salida 
    println("El area del triangulo es:"+area) 
  } 
}